examples/specs/geo_text.vl.json

```diff
@@ -36,9 +32,22 @@
         "y": {
           "field": "lat",
           "type": "latitude"
+        }
+      },
+      "layer": [{
+        "mark": "circle",
```

Comment:
Why not using color in mark?  (more concise)

Reply:
Will do when I update the spec for you lat/long PR.